### DISCUSSION TITLE
deprecated NumberEntity

### DISCUSSION THREAD

**Discussion by pabe40** [wills106/homeassistant-solax-modbus#315] [Q&A] [answered]
I'm getting this message in the logs:

WARNING (MainThread) [homeassistant.components.number] Entity None (<class 'custom_components.solax_modbus.number.SolaXModbusNumber'>) is using deprecated NumberEntity features which will be unsupported from Home Assistant Core 2022.10, please report it to the custom integration author.

Is this problematic?
Could this be related to a problem i experience when writing a value to a register ( eg. number.solax_battery_discharge_max_current ).
Seems I can't persist the value in that register.

**Comment by wills106** [comment, marked as answer]
What version of this Integration are you using?

NumberEntity was updated to comply with 'Home Assistant Core 2022.10' back in July 2022

**Reply by pabe40** [reply]
Updated HA integration homeassistant-solax-modbus . Message is gone. Thanks for the hint.
